multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +39,29 @@ coreos:
       command: start
       enable: true
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=load-rkt-stage1.service
```

Comment:
Do we want to do this every time kubelet starts? Or should we just opportunistically do it once at boot?